contrib/ovirt/lib/ovirtlago/testlib.py

```diff
@@ -138,7 +138,7 @@ def addError(self, test, err):
     def addFailure(self, test, err):
         self._addFault(test, err)
 
-    def _addFault(self, test, err):
```

Comment:
it's used just in the line before the def, you should change it there too (and any other place it's called if any)